### DISCUSSION TITLE
Counterparty Teleburn

### DISCUSSION THREAD

**Discussion by dannydeezy** [ordinals/ord#2863] [Ideas]
Lots of digital artifact culture on bitcoin traces back to Counterparty (XCP). It would be cool if we streamlined the path for those folks to transition assets into Ordinals. Perhaps a counterparty Teleburn? Is there something relevant here with Ordinal Envelopes? https://x.com/wasthatawolf/status/1632829925181861895?s=20

**Comment by raphjaph** [comment]
Would definitely be cool to add more chains. We can use the same scheme as for ethereum, just creating a counterparty address from the inscription id
